src/scenarios/scenarios_3.ts

```diff
@@ -1,12 +1,52 @@
 import {SubstrateService} from "../substrate-service";
 import {ScenarioInterface} from "./scenario-interface";
+import API from '@polkadot/api';
 
 export class Scenarios_3 implements ScenarioInterface {
   async run() {
     /*
        TODO: Implement Scenario 3 [Batch processing]:
          * Create 3 user wallets
-         * Sends assets to those wallets using batch transaction using link: https://github.com/Cerebellum-Network/private-standalone-network-node/blob/feature/fee-abstraction-doc/docs/batch_processing.md#2-using-polkadot-utilitybatch-method
+         * Sends assets to those wallets using batch transaction using link: https://github.com/Cerebellum-Network/private-standalone-network-node/blob/dev/docs/batch_processing.md
      */
+    console.log('[MAIN] Creating SubstrateService instance scenario 3...');
+
+    const service = new SubstrateService();
+    await service.initialize();
+    console.log(`[MAIN] SubstrateService instance has been created and initialized successfully`);
+
+    // Generating Wallets
+    console.log('[MAIN] Generating wallet...');
+    const apple = service.generateWallet();
+    console.log(`[MAIN] Wallet has been generated successfully. Value is ${JSON.stringify(apple)}`);
+
+    console.log('[MAIN] Generating wallet...');
+    const orange = service.generateWallet();
+    console.log(`[MAIN] Wallet has been generated successfully. Value is ${JSON.stringify(orange)}`);
+
+    console.log('[MAIN] Generating wallet...');
+    const banana = service.generateWallet();
+    console.log(`[MAIN] Wallet has been generated successfully. Value is ${JSON.stringify(banana)}`);
+
+    // Batch transactions
+    console.log(`[MAIN] Issuing assets to the user...`);
+    const result = await service.issueAssetToUserBatch(
+      apple.publicKey,
+      process.env.AMOUNT,
```

Comment:
Let's use `SCENARIO_3_AMOUN` here.

Reply:
changed this to only use one standard AMOUNT. No need to have multiple amounts